<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/Netflix-Premium-Gratis-en-2022-Apk-e1659877674993.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>341</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    background-color: rgba(0, 0, 0, 150); /* nền tối mờ */
    border-radius: 10px;</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
    font-size: 32px;
    font-weight: bold;
    letter-spacing: 1px;
 background-color: rgba(0, 0, 0, 150); /* nền tối mờ */</string>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    border: 1px solid #555555;
    border-radius: 5px;
    padding: 10px;
    font-size: 16px;</string>
    </property>
    <property name="placeholderText">
     <string>email or phone number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_password">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    border: 1px solid #555555;
    border-radius: 5px;
    padding: 10px;
    font-size: 16px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e50914; /* Màu đỏ đặc trưng */
    color: white; /* Màu chữ trắng */
    border: none; /* Không có viền */
    padding: 10px 20px; /* Khoảng cách bên trong */
    font-size: 16px; /* Cỡ chữ */
    border-radius: 4px; /* Bo góc nhẹ */</string>
    </property>
    <property name="text">
     <string>Sign in</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
    font-size: 14px;
    font-family: Arial, sans-serif;</string>
    </property>
    <property name="text">
     <string>Remember me</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
    font-size: 14px;
    text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Need help?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eye">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/eye-slash-solid.svg</normaloff>../img/eye-slash-solid.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #e50914; /* Màu đỏ đặc trưng */
    color: white; /* Màu chữ trắng */
    border: none; /* Không có viền */
    padding: 10px 20px; /* Khoảng cách bên trong */
    font-size: 16px; /* Cỡ chữ */
    border-radius: 4px; /* Bo góc nhẹ */</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
